pandas/io/tests/test_packers.py

```diff
@@ -333,7 +333,8 @@ def setUp(self):
             'E': [0., 1, Timestamp('20100101'), 'foo', 2.],
             'F': [Timestamp('20130102', tz='US/Eastern')] * 2 +
                  [Timestamp('20130603', tz='CET')] * 3,
-            'G': [Timestamp('20130102', tz='US/Eastern')] * 5
+            'G': [Timestamp('20130102', tz='US/Eastern')] * 5,
+            'H': Categorical(['a', 'b', 'c', 'd', 'e'])
```

Comment:
add in another ``Categorical`` with an ``ordered=True``. Further add one with datetimes as the categories